USERMANUAL.md

```diff
@@ -604,3 +604,14 @@ CloudDQ supports the follow methods for authenticating to GCP:
 1. OAuth via locally discovered [Application Default Credentials (ADC)](https://cloud.google.com/docs/authentication/best-practices-applications#overview_of_application_default_credentials) if only the arguments `--gcp_project_id`, `--gcp_bq_dataset_id`, and `--gcp_region_id` are provided.
 2. Using an [exported JSON service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) if the argument `--gcp_service_account_key_path` is provided.
 3. [Service Account impersonation](https://cloud.google.com/iam/docs/impersonating-service-accounts) via a source credentials if the argument `--gcp_impersonation_credentials` is provided. The source credentials can be obtained from either `--gcp_service_account_key_path` or from the local ADC credentials.
+
+### Materialization Hours
+1. Configure the bigquery job intermediate tables expiration hours in the provided dataset `--gcp_bq_dataset_id`.
+2. if the CLI argument `--intermediate_table_expiration_hours` is defined, CloudDQ will set the same as expiration hours for intermediate tables. 
+  `--intermediate_table_expiration_hours` is currently an optional argument and has a default value of `24 hours`.
```

Comment:
How about: The default "expiration hours" for BigQuery job intermediate tables in the provided dataset `--gcp_bq_dataset_id` is set to `24 hours`. If you'd like to change it, ... ...

Reply:
update the doc